### PR TITLE
gpclient: add page

### LINKED ISSUE #14034: Page request: gpclient
### Command description

Client for using Globalprotect VPN

### Command details

_No response_

### Documentation

https://github.com/yuezk/GlobalProtect-openconnect

### Platform

Linux

### VCS repository link (e.g. GitHub, GitLab)

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by KrishDave1**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):** 

Closes #14034

### CODE REVIEW

File: pages/linux/gpclient.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# gpclient
+
+> Client for using GlobalProtect VPN on Linux via OpenConnect.
+> `gpclient` allows you to connect, disconnect, and manage VPN connections using a command-line interface or a graphical user interface (GUI).
```
Comment (Managor) [inline pages/linux/gpclient.md:4]:
```suggestion
> Connect to a GlobalProtect VPN on Linux via OpenConnect.
```
Keep the description as brief as possible. The commands will document themselves. If possible, use imperative mood.

File: pages/linux/gpclient.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+- Ignore TLS errors during connection:
+
+`gpclient connect --ignore-tls-errors {{vpn_gateway_url}}`
+
+- Print version information:
+
+`gpclient --version`
+
+- Print help for any command:
```
Comment (Managor) [inline pages/linux/gpclient.md:31]:
```suggestion
- Display help for any command:
```
We have a convention to use "Display help" for help pages.

**Review by Managor — COMMENTED** [review]
Thank you for your first contribution

**Comment by KrishDave1** [comment]
Hey @Managor , I have applied the changes you suggested. Please review it.

**Review by cyqsimon — APPROVED** [review]

**Review by Managor — APPROVED** [review]

File: pages/linux/gpclient.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+- Use OpenSSL workaround to bypass legacy renegotiation errors:
+
+`gpclient connect --fix-openssl {{vpn_gateway_url}}`
+
+- Ignore TLS errors during connection:
+
+`gpclient connect --ignore-tls-errors {{vpn_gateway_url}}`
+
+- Print version information:
```
Comment (spageektti) [inline pages/linux/gpclient.md:26]:
```suggestion
- Display version:
```
see style guide

File: pages/linux/gpclient.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+`gpclient --version`
+
+- Display help for any command:
```
Comment (spageektti) [inline pages/linux/gpclient.md:30]:
```suggestion
- Display help:
```

**Review by spageektti — COMMENTED** [review]
Hi. Thanks for your contribution.

**Review by spageektti — APPROVED** [review]

**Pull request merged**